src/Compilers/Core/Portable/DiagnosticAnalyzer/DictionaryAnalyzerConfigOptions.cs

```diff
@@ -13,12 +14,17 @@ internal sealed class DictionaryAnalyzerConfigOptions : AnalyzerConfigOptions
 
         public static DictionaryAnalyzerConfigOptions Empty { get; } = new DictionaryAnalyzerConfigOptions(EmptyDictionary);
 
+        // Note: Do not rename. Older versions of analyzers access this field via reflection.
+        // https://github.com/dotnet/roslyn/blob/8e3d62a30b833631baaa4e84c5892298f16a8c9e/src/Workspaces/SharedUtilitiesAndExtensions/Compiler/Core/Options/EditorConfig/EditorConfigStorageLocationExtensions.cs#L21
         internal readonly ImmutableDictionary<string, string> Options;
```

Comment:
💡 I've seen comments get overlooked/deleted. Consider also adding a unit test that accesses this field via reflection (and specifically doesn't use `nameof`).